features/enlinkd/adapters/collectors/ipnettomedia/src/main/java/org/opennms/netmgt/enlinkd/NodeDiscoveryIpNetToMedia.java

```diff
@@ -80,47 +80,31 @@ protected void runNodeDiscovery() {
         IpNetToMediaTableTracker ipNetToMediaTableTracker = new IpNetToMediaTableTracker() {
             public void processIpNetToMediaRow(final IpNetToMediaRow row) {
                 IpNetToMedia macep = row.getIpNetToMedia();
-                if (macep.getPhysAddress() != null && 
-                        macep.getNetAddress() != null && 
-                        macep.getSourceIfIndex() != null &&
-                        (macep.getIpNetToMediaType() == IpNetToMediaType.IPNETTOMEDIA_TYPE_DYNAMIC
-                        || macep.getIpNetToMediaType() == IpNetToMediaType.IPNETTOMEDIA_TYPE_STATIC)) {
-                    if (LOG.isDebugEnabled()) {
-                        LOG.debug("processIpNetToMediaRow: node [{}], mac address {} and ip {} mediatype {}. saving",
+
+                LOG.debug("processIpNetToMediaRow: node [{}], {}:{}:{}. ",
+                          getNodeId(),
+                          macep.getPhysAddress(),
+                          str(macep.getNetAddress()),
+                          macep.getIpNetToMediaType());
+
+                if (macep.getPhysAddress() == null
+                        || macep.getNetAddress() == null
+                        || macep.getIpNetToMediaType() == null
+                        || macep.getIpNetToMediaType() == IpNetToMediaType.IPNETTOMEDIA_TYPE_INVALID) {
+                    LOG.debug("processIpNetToMediaRow: node [{}], {}:{}:{}. skipping",
                               getNodeId(),
-                              macep.getPhysAddress(), 
+                              macep.getPhysAddress(),
                               str(macep.getNetAddress()),
```

Comment:
Similarly here.

Reply:
well now I have set in this class the if LOG.isDebugEnabled...so no need to revert changes...